nb.rs,"HTTPSConnectionPool(host='nb.rs', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='nb.rs', port=80): Read timed out. (read timeout=2)"
ncc.gov.ng,"HTTPSConnectionPool(host='ncc.gov.ng', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='ncc.gov.ng', port=443): Read timed out. (read timeout=2)"
neik.vn,"HTTPSConnectionPool(host='neik.vn', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='neik.vn', port=443): Read timed out. (read timeout=2)"
neliti.com,"HTTPSConnectionPool(host='neliti.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.neliti.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
neopod.ru,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","HTTPConnectionPool(host='neopod.ru', port=80): Read timed out. (read timeout=2)"
neowing.co.jp,"HTTPSConnectionPool(host='neowing.co.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f7b968b9460>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='neowing.co.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f7b94dd56a0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
nestia.com,"HTTPSConnectionPool(host='www.nestia.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.nestia.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1124)')))"
net.ee,"HTTPSConnectionPool(host='net.ee', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f7b962fb160>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='net.ee', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f7b95302250>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
netky.sk,"HTTPSConnectionPool(host='netky.sk', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f7b95ecedc0>, 'Connection to netky.sk timed out. (connect timeout=2)'))","HTTPConnectionPool(host='www.netky.sk', port=80): Read timed out. (read timeout=2)"
neturuguay.com,"HTTPSConnectionPool(host='neturuguay.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='neturuguay.com', port=443): Read timed out. (read timeout=2)"
netwerkgalerij.be,"HTTPSConnectionPool(host='netwerkgalerij.be', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f7b9578b820>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='netwerkgalerij.be', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f7b91d5cac0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
newedc.com,"HTTPSConnectionPool(host='www.newedc.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.newedc.com', port=443): Read timed out. (read timeout=2)"
newellwind.com,"HTTPSConnectionPool(host='newellwind.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f7b952e02b0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='newellwind.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f7b93efb760>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
newpages.com.my,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","HTTPSConnectionPool(host='www.newpages.com.my', port=443): Read timed out. (read timeout=2)"
news-italia-mondo.it,"HTTPSConnectionPool(host='news-italia-mondo.it', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='news-italia-mondo.it', port=80): Read timed out. (read timeout=2)"
newsformy.com,"HTTPSConnectionPool(host='newsformy.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='newsformy.com', port=443): Read timed out. (read timeout=2)"
newsplus.co.th,"HTTPSConnectionPool(host='newsplus.co.th', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='newsplus.co.th', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f7b90a5ba90>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
newswire.ca,"HTTPSConnectionPool(host='newswire.ca', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))","HTTPSConnectionPool(host='newswire.ca', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))"
newtv.co.th,"HTTPSConnectionPool(host='newtv.co.th', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='newtv.co.th', port=80): Read timed out. (read timeout=2)"
ngobg.info,"HTTPSConnectionPool(host='www.ngobg.info', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.ngobg.info', port=443): Read timed out. (read timeout=2)"
nguoiduatin.vn,"HTTPSConnectionPool(host='nguoiduatin.vn', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f7b9545b820>, 'Connection to nguoiduatin.vn timed out. (connect timeout=2)'))","HTTPConnectionPool(host='nguoiduatin.vn', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f7b91ec85b0>, 'Connection to nguoiduatin.vn timed out. (connect timeout=2)'))"
nhk.or.jp,"HTTPSConnectionPool(host='nhk.or.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f7b95acf340>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='nhk.or.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f7b94bfa250>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
ni.rs,"HTTPSConnectionPool(host='ni.rs', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f7b93119c70>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='ni.rs', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f7b953a8bb0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
nihon-kankou.or.jp,"HTTPSConnectionPool(host='nihon-kankou.or.jp', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.nihon-kankou.or.jp', port=443): Read timed out. (read timeout=2)"
nii.ac.jp,"HTTPSConnectionPool(host='nii.ac.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f7b97e0acd0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='nii.ac.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f7b8f7adcd0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
niigata.lg.jp,"HTTPSConnectionPool(host='niigata.lg.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f7b96758d60>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='niigata.lg.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f7b900d3880>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
nikkei.co.jp,"HTTPSConnectionPool(host='nikkei.co.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f7b8de86850>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='nikkei.co.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f7b939ec160>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
nilephotos.net,"HTTPSConnectionPool(host='nilephotos.net', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='nilephotos.net', port=443): Read timed out. (read timeout=2)"
ninhbinh.gov.vn,"HTTPSConnectionPool(host='ninhbinh.gov.vn', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='ninhbinh.gov.vn', port=443): Read timed out. (read timeout=2)"
ninhthuan.gov.vn,"HTTPConnectionPool(host='www.ninhthuan.gov.vn', port=80): Read timed out. (read timeout=2)","HTTPConnectionPool(host='www.ninhthuan.gov.vn', port=80): Read timed out. (read timeout=2)"
nippon.com,"HTTPSConnectionPool(host='nippon.com', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f7b959ece50>, 'Connection to nippon.com timed out. (connect timeout=2)'))","HTTPConnectionPool(host='nippon.com', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f7b8ddd7c40>, 'Connection to nippon.com timed out. (connect timeout=2)'))"
njuzz.ee,"HTTPSConnectionPool(host='njuzz.ee', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f7b9653d2e0>, 'Connection to njuzz.ee timed out. (connect timeout=2)'))","HTTPConnectionPool(host='njuzz.ee', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f7b965a4910>: Failed to establish a new connection: [Errno 113] No route to host'))"
nojima.co.jp,"HTTPSConnectionPool(host='nojima.co.jp', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f7b8dddb370>, 'Connection to nojima.co.jp timed out. (connect timeout=2)'))","HTTPConnectionPool(host='nojima.co.jp', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f7b919723a0>, 'Connection to nojima.co.jp timed out. (connect timeout=2)'))"
nomos-elibrary.de,"HTTPSConnectionPool(host='www.nomos-elibrary.de', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.nomos-elibrary.de', port=443): Read timed out. (read timeout=2)"
novine.hr,"HTTPSConnectionPool(host='novine.hr', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='novine.hr', port=80): Read timed out. (read timeout=2)"
novosti.hr,"HTTPSConnectionPool(host='novosti.hr', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='novosti.hr', port=443): Read timed out. (read timeout=2)"
npru.ac.th,"HTTPSConnectionPool(host='npru.ac.th', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.npru.ac.th', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
npz.sk,"HTTPSConnectionPool(host='npz.sk', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1124)')))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
nrsr.sk,"HTTPSConnectionPool(host='nrsr.sk', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f7b960b80a0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='nrsr.sk', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f7b91bb3d30>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
ns.ac.rs,"HTTPSConnectionPool(host='ns.ac.rs', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f7b94aad790>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='ns.ac.rs', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f7b9560e760>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
nsc.ru,"HTTPSConnectionPool(host='nsc.ru', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f7b8ecfeac0>, 'Connection to nsc.ru timed out. (connect timeout=2)'))","HTTPConnectionPool(host='nsc.ru', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f7b93db4ee0>, 'Connection to nsc.ru timed out. (connect timeout=2)'))"
nsk.hr,"HTTPSConnectionPool(host='nsk.hr', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='nsk.hr', port=443): Read timed out. (read timeout=2)"
nso.go.th,"HTTPSConnectionPool(host='nso.go.th', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f7b900a21c0>, 'Connection to nso.go.th timed out. (connect timeout=2)'))","HTTPConnectionPool(host='nso.go.th', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f7b8dfa7310>, 'Connection to nso.go.th timed out. (connect timeout=2)'))"
nstru.ac.th,"HTTPSConnectionPool(host='nstru.ac.th', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f7b9815d550>: Failed to establish a new connection: [Errno 101] Network is unreachable'))","HTTPConnectionPool(host='nstru.ac.th', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f7b8f7b6760>: Failed to establish a new connection: [Errno 101] Network is unreachable'))"
nta.go.jp,"HTTPSConnectionPool(host='nta.go.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f7b9002f970>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='nta.go.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f7b8f77ba60>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
nttdocomo.co.jp,"HTTPSConnectionPool(host='nttdocomo.co.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f7b8e2c1250>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='nttdocomo.co.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f7b94cb8d90>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
ntua.gr,"HTTPSConnectionPool(host='ntua.gr', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f7b8e07a490>: Failed to establish a new connection: [Errno 101] Network is unreachable'))","HTTPConnectionPool(host='ntua.gr', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f7b8e0114f0>: Failed to establish a new connection: [Errno 101] Network is unreachable'))"
number1.co.il,"HTTPSConnectionPool(host='number1.co.il', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='number1.co.il', port=80): Read timed out. (read timeout=2)"
numo.pl,"HTTPSConnectionPool(host='www.multiprojektowanie.pl', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f7b96e33400>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPSConnectionPool(host='www.multiprojektowanie.pl', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f7b8e0a8a90>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
nusalist.com,"HTTPSConnectionPool(host='nearbymalaysia.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='nearbymalaysia.com', port=443): Read timed out. (read timeout=2)"
nuv.cz,"HTTPSConnectionPool(host='nuv.cz', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: UNSUPPORTED_PROTOCOL] unsupported protocol (_ssl.c:1124)')))","HTTPConnectionPool(host='www.nuv.cz', port=80): Read timed out. (read timeout=2)"
nwu.ac.za,"HTTPSConnectionPool(host='nwu.ac.za', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f7b938a0310>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='nwu.ac.za', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f7b8e0b1910>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
nyf.hu,"HTTPSConnectionPool(host='nyf.hu', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f7b95008c40>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='nyf.hu', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f7b95a8e280>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
nyitvatartas24.hu,"HTTPSConnectionPool(host='nyitvatartas24.hu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.nyitvatartas24.hu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))"
nylon.com.sg,"HTTPSConnectionPool(host='www.nylon.com.sg', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.nylon.com.sg', port=443): Read timed out. (read timeout=2)"
oae.go.th,"HTTPSConnectionPool(host='oae.go.th', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f7b96514760>: Failed to establish a new connection: [Errno 101] Network is unreachable'))","HTTPConnectionPool(host='oae.go.th', port=80): Read timed out. (read timeout=2)"
oaed.gr,"HTTPSConnectionPool(host='oaed.gr', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='oaed.gr', port=443): Read timed out. (read timeout=2)"
oamk.fi,"HTTPSConnectionPool(host='www.oamk.fi', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.oamk.fi', port=443): Read timed out. (read timeout=2)"
obraspublicas.gob.ec,"HTTPSConnectionPool(host='obraspublicas.gob.ec', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'obraspublicas.gob.ec' doesn't match either of 'www.agricultura.gob.ec', 'agricultura.gob.ec'"")))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
obz.hr,"HTTPSConnectionPool(host='obz.hr', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f7b8d897490>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='obz.hr', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f7b91f27250>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
oca.com.uy,"HTTPSConnectionPool(host='oca.com.uy', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f7b96447370>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='oca.com.uy', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f7b91f27370>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
